src/systemdhook.c

```diff
@@ -822,14 +822,19 @@ int main(int argc, char *argv[])
 		snprintf(config_file_name, PATH_MAX, "%s/config.json", YAJL_GET_STRING(v_bundle_path));
 		fp = fopen(config_file_name, "r");
 	} else {
-		char msg[] = "bundlePath not found in state";
+		char msg[] = "bundlePath not found";
 		snprintf(config_file_name, PATH_MAX, "%s", msg);
+		pr_pinfo("Failed reading state data: %s.  Generally this indicates Docker versions prior to 1.12 are installed.", config_file_name);
```

Comment:
You know, that's probably a good idea.  I think I'll add an EXIT_SUCCESS here then change the below EXIT_SUCCESS at line 836 back to an EXIT_ERROR.  Good thought.